data/spells/scripts/support/find_person.lua

```diff
@@ -46,6 +46,7 @@ function onCastSpell(creature, variant)
 	end
 
 	local playerPosition = creature:getPosition()
+	local targetPosition = targetPlayer:getPosition()
```

Comment:
You should fix one more thing, i'd point out in your last pr, however i didn't make before it gets merged. In this case variant.text is nill, to variant.text be filled this statement must be false, which is not because it needs a target, instead you should get variant.number which store player id, also i think player shouldn't be able to exiva himself.